static/app/views/routeNotFound.tsx

```diff
@@ -8,11 +8,13 @@ import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import Sidebar from 'sentry/components/sidebar';
 import {t} from 'sentry/locale';
 import type {RouteComponentProps} from 'sentry/types/legacyReactRouter';
+import {LastKnownRouteContext} from 'sentry/views/lastKnownRouteContextProvider';
 
 type Props = RouteComponentProps<{}, {}>;
 
 function RouteNotFound({router, location}: Props) {
   const {pathname, search, hash} = location;
+  const lastRoute = useContext(LastKnownRouteContext);
```

Comment:
Let's change like this
```suggestion
  const lastRoute = useLastKnownRoute();
```